tests/components/Text.spec.tsx

```diff
@@ -9,23 +9,20 @@ describe('tests in <Text/>', () => {
     const content = "I'm a paragraph";
     const variant = 'md';
 
-    const { container } = render(<Text variant={variant} color={colors.purple[500]}>{content}</Text>)
+    const { getByText } = render(<Text variant={variant} color={colors.purple[500]}>{content}</Text>)
 
-    const text = container.querySelector('p') as HTMLParagraphElement;
+    const text = getByText(content) as HTMLParagraphElement;
     expect(text).toBeTruthy();
   })
 
   test('should render text with a variant', () => { 
     const content = "I'm a h1 text";
     const variant = 'h1';
 
-    const { container } = render(<Text variant={variant} color={colors.purple[500]}>{content}</Text>)
+    const { getByRole } = render(<Text variant={variant} color={colors.purple[500]}>{content}</Text>)
```

Comment:
¿Hay alguna ventaja sobre este approach en vez de usar `screen.getByRole`?

https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen